services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050345  | Regional Load Balancing                               | Load balancers can be distributed across multiple zones in the same region.                                                                                                                                                                                                                                                   |
+| CCC-050346  | Cross-Regional Load Balancing                         | Load balancers can be distributed across multiple regions.                                                                                                                                                                                                                                                                    |
+| CCC-050347  | Flow stickiness                                       | Allows requests from the same client to be consistently routed to the same backend server during the duration of a session.                                                                                                                                                                                                   |
+| CCC-050348  | Automatic failover to healthy backends                | Detect and redirect traffic away from unhealthy or failed backend servers to healthy ones automatically.                                                                                                                                                                                                                      |
+| CCC-050349  | Configurable standby backends                         | Configure backend targets that are only routed to if primary targets become unhealthy.                                                                                                                                                                                                                                        |
+| CCC-050350  | Connection draining on failover and failback          | Allows existing TCP sessions to remain active for a configurable time period even after backend VMs become unhealthy.                                                                                                                                                                                                         |
+| CCC-050351  | Health Checks                                         | Can perform health checks by sending HTTP, HTTPS, TCP, or SSL requests to specified endpoints on backend instances and evaluate the responses to determine the health status.                                                                                                                                                 |
```

Comment:
LBs should provide Health Check endpoints for them selves if I'm not mistaken. That should be a feature of LB as well. Can you check this. Then when you have NLB in front of set of LBs, if the LB itself is not healthy, it can omit that from its targets list.